objc: test fails on MacBook M3 Pro 2023

```
% git switch main
Switched to branch 'main'
Your branch is up to date with 'origin/main'.
hajimehoshi@Hajimes-MacBook-Pro purego % go test ./...
?       github.com/ebitengine/purego/examples/libc      [no test files]
?       github.com/ebitengine/purego/examples/objc      [no test files]
?       github.com/ebitengine/purego/internal/buildtest [no test files]
?       github.com/ebitengine/purego/internal/cgo       [no test files]
?       github.com/ebitengine/purego/internal/fakecgo   [no test files]
?       github.com/ebitengine/purego/internal/strings   [no test files]
ok      github.com/ebitengine/purego    1.136s
--- FAIL: TestSend (0.00s)
    objc_runtime_darwin_test.go:109: dlopen(Foundation.framework/Foundation, 0x0008): tried: 'Foundation.framework/Foundation' (no such file), '/System/Volumes/Preboot/Cryptexes/OSFoundation.framework/Foundation' (no such file), '/usr/lib/Foundation.framework/Foundation' (no such file, not in dyld cache), 'Foundation.framework/Foundation' (no such file)
FAIL
FAIL    github.com/ebitengine/purego/objc       0.427s
FAIL
```

My guess is that we have to specify a fullpath for a framework.

My machine is:

```
                    'c.          hajimehoshi@Hajimes-MacBook-Pro.local 
                 ,xNMM.          ------------------------------------- 
               .OMMMMo           OS: macOS 14.2.1 23C71 arm64 
               OMMM0,            Host: Mac15,6 
     .;loddo:' loolloddol;.      Kernel: 23.2.0 
   cKMMMMMMMMMMNWMMMMMMMMMM0:    Uptime: 2 days, 21 hours, 34 mins 
 .KMMMMMMMMMMMMMMMMMMMMMMMWd.    Packages: 24 (brew) 
 XMMMMMMMMMMMMMMMMMMMMMMMX.      Shell: zsh 5.9 
;MMMMMMMMMMMMMMMMMMMMMMMM:       Resolution: 1800x1169 
:MMMMMMMMMMMMMMMMMMMMMMMM:       DE: Aqua 
.MMMMMMMMMMMMMMMMMMMMMMMMX.      WM: Quartz Compositor 
 kMMMMMMMMMMMMMMMMMMMMMMMMWd.    WM Theme: Blue (Light) 
 .XMMMMMMMMMMMMMMMMMMMMMMMMMMk   Terminal: tmux 
  .XMMMMMMMMMMMMMMMMMMMMMMMMK.   CPU: Apple M3 Pro 
    kMMMMMMMMMMMMMMMMMMMMMMd     GPU: Apple M3 Pro 
     ;KMMMMMMMWXXWMMMMMMMk.      Memory: 3437MiB / 18432MiB 
       .cooc,.    .,coo:.
                                                         
                                                         


```

--- a/objc/objc_runtime_darwin_test.go
+++ b/objc/objc_runtime_darwin_test.go
@@ -104,7 +104,7 @@ func ExampleID_SendSuper() {
 
 func TestSend(t *testing.T) {
 	// NSNumber comes from Foundation so make sure we have linked to that framework.
-	_, err := purego.Dlopen("Foundation.framework/Foundation", purego.RTLD_GLOBAL)
+	_, err := purego.Dlopen("/System/Library/Frameworks/Foundation.framework/Foundation", purego.RTLD_GLOBAL)
 	if err != nil {
 		t.Fatal(err)
 	}

